staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -532,6 +532,8 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 		UnsafeConvertor: a.group.UnsafeConvertor,
 		Authorizer:      a.group.Authorizer,
 
```

Comment:
why empty line? Is that different?

Reply:
just to avoid big diffs on aligned fields